openmc/deplete/results_list.py

```diff
@@ -192,3 +194,78 @@ def get_depletion_time(self):
         for ix, res in enumerate(items):
             times[ix] = res.proc_time
         return times
+
+    def export_to_materials(self, burnup_index, nuc_with_data=None):
+        """Converts a depletion time point composition into an openmc.Materials
+        instance, useful for running transport calculations on individual
+        depletion time points
+
+        This requires a materials.xml file to be present in the current
+        directory. This should correspond to the materials initially
+        present in the depletion calculation.
```

Comment:
[numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) indicates the first line of a function / method docstring should be a complete sentence, e.g.,
```suggestion
        """Create an openmc.Materials with compositions from a depletion point.
        
        This is useful for running transport calculations on individual
        depletion time points. This requires a materials.xml file to be present in 
        the current directory. This should correspond to the material initially
        present in the depletion calculation.
```